### PR TITLE
Added Rows package and 'rows.map' function

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/reference/flux/stdlib/contrib/rows/map.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+    parent: Rows
+weight: 202
+v2.0/tags: [functions, package, map]
+related:
+  - /v2.0/reference/flux/stdlib/built-in/transformations/map/
+---
+
+The `rows.map()` function is an alternate implementation of [`map()`](/v2.0/reference/flux/stdlib/built-in/transformations/map/)
+that is faster, but more limited than `map()`.
+`rows.map()` cannot modify table groups key and, therefore, does not need to regroup tables.
```
Comment (noramullen1) [inline content/v2.0/reference/flux/stdlib/contrib/rows/map.md:18]: table group keys? Unless this is an "attorneys general" situation I'm not understanding haha.

**Comment by jsternberg** [comment]
This was a contrib function so it isn't a formal addition. I don't know how to currently deal with contrib, but if we document them, it needs to be clearly labeled as not part of the stdlib.

**Comment by jsternberg** [comment]
The documentation looks good though.

File: content/v2.0/reference/flux/stdlib/contrib/rows/map.md (hunk 1 above)
Comment (sanderson) [reply content/v2.0/reference/flux/stdlib/contrib/rows/map.md]: I removed "table" and added a link to the "group key" definition in the glossary.

**Pull request merged**